src/iop/temperature.c

```diff
@@ -223,12 +223,12 @@ void init_key_accels(dt_iop_module_so_t *self)
   dt_accel_register_slider_iop(self, FALSE, NC_("accel", "green"));
   dt_accel_register_slider_iop(self, FALSE, NC_("accel", "blue"));
   dt_accel_register_slider_iop(self, FALSE, NC_("accel", "emerald"));
-  dt_accel_register_combobox_iop(self, FALSE, NC_("accel", "presets"));
+  dt_accel_register_combobox_iop(self, FALSE, NC_("accel", "settings"));
 
-  dt_accel_register_iop(self, FALSE, NC_("accel", "preset/as shot"), 0, 0);
-  dt_accel_register_iop(self, FALSE, NC_("accel", "preset/camera reference"), 0, 0);
-  dt_accel_register_iop(self, FALSE, NC_("accel", "preset/from image area"), 0, 0);
-  dt_accel_register_iop(self, FALSE, NC_("accel", "preset/user modified"), 0, 0);
+  dt_accel_register_iop(self, FALSE, NC_("accel", "settings/as shot"), 0, 0);
+  dt_accel_register_iop(self, FALSE, NC_("accel", "settings/from image area"), 0, 0);
+  dt_accel_register_iop(self, FALSE, NC_("accel", "settings/user modified"), 0, 0);
+  dt_accel_register_iop(self, FALSE, NC_("accel", "settings/camera reference"), 0, 0);
```

Comment:
Technicalllyyyy... one might be noisy about it but grouping direct access to settings with shortcuts to settings might look weird. When you open up shortcuts it's like:

settings:
- as shot
- camera reference
- dynamic
- from image area
- next
- previous
- user modifed

It might be absolute nitpicking but separating that into
setting -> [shortcut to predefined setting]
settings -> [combo box handling]
would make this 100% awesome ;)

Reply:
I actually think this is great and as it should be; all six of them are about choosing a setting in the drop-down...
Having two groups that actually manipulate the same thing, as before, looked wrong to me.

But if I haven't convinced you, I'll gladly change it back; this is all about making you happy!! After all the work you put in, I don't want you to feel the end result isn't _exactly_ what you wanted.

Reply:
> I actually think this is great and as it should be

Then it should be that way! all 6 in one place :)

So this is perfect as is!